README.md

```diff
@@ -40,6 +26,27 @@ Even if a new feature is not listed yet, please open an issue.
 | Talk          | :rocket:           |
 | Tasks         | :rocket:           |
 
+## Why this framework exists aka. problems with other clients
```

Comment:
This mainly describes reasons why you want to use flutter but not really neon specific.

I think a big problem is that many NC apps (php) exist and NC (the company) isn't that much involved in mobile clients. This lead to the community writing their own clients (most of the time ios and android) where sepparate projects and efforts emerged.

This means that the iOS app, the android app and the php app didn't have anything in common.

This project tries to solve this:
1. generated openapi specs
2. flutter for the clients
3. the main reason for me would be that neon enables me to not implement everything from scratch (loginflow, api,caching, ...)

this is somehow what `#Architecture overview` tells but I think those reasons could also be mentioned

Reply:
You are right, I just copied it from my job application, that's why I didn't went straight into the technical details.

The generated OpenAPI specs originate from this project, but are now part of my work at the Nextcloud company. Of course this project massively benefits from that, but I don't really consider them part of this project, since the tools I built for it are also not part of this project.

I will add the other reasons you are giving. The current list is really just my thoughts back then, but of course they have somewhat changed.

Reply:
Yeah I mainly meant that this project reduces the cost of starting a NC client app imensly.

And the openapi specs are a big reason it's so easy for neon.

Reply:
I added some more text about this. I think my explanation for OpenAPI is good, but the other things are not that well explain. Do you have some suggestions on how to make it better?

Reply:
> Trying to make everything re-usable is a major goal of this project, as we want to close the gap between platforms and functionalities and make development of new clients and apps enjoyable and fast.

We could add something like 'developing a new client can be as easy as providing the your UI' at least that's what neon is to me (and cookbook). 
Write a Bloc and add some UI :)

Reply:
But I also quite like the current wording.

Reply:
Good idea, will add it.